src/java/com/google/tagpost/TagpostClient.java

```diff
@@ -0,0 +1,65 @@
+package com.google.tagpost;
+
+import com.google.tagpost.FetchMessageRequest;
+import com.google.tagpost.FetchMessageResponse;
+import com.google.tagpost.TagpostServiceGrpc;
+import io.grpc.Channel;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class TagpostClient {
+  private static final Logger logger = Logger.getLogger(TagpostClient.class.getName());
```

Comment:
Suggestion: If you have time, try to use Google Flogger. https://google.github.io/flogger/
You can try it in next PR.

Reply:
Flogger is being processed in another PR